### DISCUSSION TITLE
Push Benachrichtigungen nur im heimischen Netzwerk

### DISCUSSION THREAD

**Discussion by CLK140172** [thkl/hap-homematic#381] [Q&A] [answered]
Hallo, erstmal vielen Dank für dieses tolle Tool. Ich habe soweit alles ohne Probleme installiert und meine Geräte in der Home App sichtbar und steuerbar. Allerdings habe ich festgestellt, dass die eingestellten Benachrichtigungen/ Mitteilungen für Sensoren und Melder nur im heimischen Netzwerk funktionieren. Sowie ich unterwegs bin, kann ich die Geräte über die Home App steuern, allerdings erhalte ich dann keine Mitteilungen wenn ich zum Beispiel die Garage öffne.

Im voraus vielen Dank für Eure Hilfe

**Comment by gerd2005** [comment]
Hallo, das liegt vermutlich daran, dass du keine HomeKit zentrale (iPad, HomePod oder Apple TV) definiert hast. Kann das sein?

**Comment by jb-home** [comment, marked as answer]
Wenn er die Geräte von unterwegs steuern kann, dann müsste er eigentlich eine HomeKit Zentrale haben.
Oder benutzt du @CLK140172 unterwegs eine VPN zum steuern?
Generell ist es aber kein HAP-HomeMatic Problem. Da die Push Mitteilungen nicht von HAP-HomeMatic kommen, sondern von HomeKit.

**Reply by ChrWi** [reply]
Kann ich bestätigen - ich habe hier keine Probleme

**Comment by CLK140172** [comment]
Ich verwende einen HomePod als Steuerzentrale und von ausserhalb ohne VPN. Dann versuche ich mal alle Geräte einschließlich HomePod neu zu starten. Vielen Dank
